### DISCUSSION TITLE
File not found error while using towhee

### DISCUSSION THREAD

**Discussion by ZubeenK** [towhee-io/towhee#2492] [Q&A]
I have added towhee==0.9.0. in my pip installation list while creating a conda environment. I am using this environment to run my script using cluster (azure cluster) in the cloud.
Let's say I am using tohwee as: 
towhee.glob['path'](<address(str)>)\
             .image_decode['path', 'img']()\

While running the script in cloud, I am getting the following error:
FileNotFoundError: [Errno 2] No such file or directory: '/root/.towhee/operators/image-decode/cv2/main/cv2.py'

How do I resolve this?

my environment looks like this:
channels:
  - anaconda
  - conda-forge
  - defaults
dependencies:
  - python=3.8
  - pip==22.1.2
  - pip:
      - azureml-core
      - azureml-defaults
      - pymilvus==2.2.1
      - towhee==0.9.0
      - opencv-python==4.7.0.72
      - pillow
      - python-certifi-win32

**Comment by junjiejiangjjj** [comment]
Does the path `/root/.towhee/operators/image-decode/cv2/main/` like this:
![image](https://user-images.githubusercontent.com/14136703/227939341-e051d6ac-bd87-49d8-9e2f-0d39bc3ae74b.png)

if so, try to install `libgl1-mesa-glx`: apt install libgl1-mesa-glx

**Reply by ZubeenK** [reply]
Yes, the path looks exactly like the image above. I will try out the method you suggested.
